UserCode/CargoPractic/AirBooking.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserCode.CargoPractic
+{
+    public class AirBooking : IBooking
+    {
+        public static AirBooking? Instance;
+        private AirBooking()
+        {
+            Instance = new AirBooking();
```

Comment:
you have to ensure that Air must be single instance. in this case we have multiple instance.

Reply:
you have to seperate the instance creation to another class (factory) to manage this.